src/redux/actions/bulkConcepts/index.js

```diff
@@ -11,18 +11,19 @@ import {
   ADD_EXISTING_BULK_CONCEPTS,
   FETCH_CONCEPT_SOURCES,
   CLEAR_SOURCE_CONCEPTS,
+  IS_LOADING,
 } from '../types';
 
 const fetchSourceConcepts = url => async (dispatch) => {
   dispatch(clear(CLEAR_SOURCE_CONCEPTS));
-  dispatch(isFetching(true));
+  dispatch(isSuccess(true, IS_LOADING));
```

Comment:
are the changes in this file necessary? . dispatching ` isFetching(true/false) ` should set `loading status` to true of false, and that can be used

Reply:
Yes, it is. There are two calls made one with the fetchSourceConcepts and the other was fetchConceptSource. Both do set loading to false once completed. There was a need to differentiate them. That was why I changed it a different name.